pinot-query-runtime/src/main/java/org/apache/pinot/query/runtime/operator/FilterOperator.java

```diff
@@ -75,7 +75,7 @@ public List<MultiStageOperator> getChildOperators() {
   @Override
   public String toExplainString() {
     _upstreamOperator.toExplainString();
-    LOGGER.debug(_operatorStats.toString());
+    LOGGER.info(_operatorStats.toString());
```

Comment:
although not introduced in this PR. but looks like there are a lot of start/stop timer within each operator on how they record the timer. Let's clean them up by recording the entire time e2e and later when logging, subtract current and upstream timer, this way 
1. all the timer operations can be done/wrapped inside `MultiStageOperator.nextBlock()` method. 
2. avoid the unnecessary start/stop on the stopwatch (although minuscule, it is still perform on a per-block basis)
3. makes the code cleaner

good example: https://github.com/apache/pinot/blob/01f3528ffc1d6db548cc38ad483bb7c061076b6a/pinot-core/src/main/java/org/apache/pinot/core/operator/BaseOperator.java#L42-L44

Reply:
This is more complicated implementation. Since we need the time now, I'll check in this PR and have followups. Thanks